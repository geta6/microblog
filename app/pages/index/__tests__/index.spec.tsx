import React from 'react';
import renderer from 'react-test-renderer';
import { IndexPage } from '..';

describe('IndexPage', () => {
  test('snap shot', () => {
    const component = renderer.create(<IndexPage />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
