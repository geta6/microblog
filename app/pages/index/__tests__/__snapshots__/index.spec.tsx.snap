// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage snap shot 1`] = `
<div
  className="style__Container-sc-1xy7lcb-0 dnIENZ"
>
  <div
    className="style__Contents-sc-1xy7lcb-1 dxkzkx"
  >
    <p>
      水馬赤いな。ア、イ、ウ、エ、オ。
    </p>
    <p>
      浮藻に子蝦もおよいでる。
    </p>
    <p>
      柿の木、栗の木。カ、キ、ク、ケ、コ。
    </p>
    <p>
      啄木鳥こつこつ、枯れけやき。
    </p>
  </div>
</div>
`;
